Automattic/newspack-plugin
# Design adjustments to the wizards

### All Submissions:






### Changes proposed in this Pull Request:

* Remove `FormattedHeader`
* Remove use of icon in the Wizard Header
* Adjust line-height and paddings
* Update components demo
* Add a noBorder const to the Card
* Update required plugin(s) screen

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the different wizards, do they work as expected?

### Other information:
